content/docs/studio/troubleshooting.md

```diff
@@ -88,3 +96,48 @@ metrics or hyperparameters that you want to visualize.
 Refer to the [DVC documentation](https://dvc.org/doc) for help on making commits
 to a DVC repository. Instructions on how to specify custom files can be found
 [here](/doc/studio/user-guide/views/view-settings#configuring-view-settings).
+
+## View does not contain the columns that I want
+
+There are two possible reasons for this:
+
+1. **The required columns were not imported:** DVC Studio will import up to 200
+   columns (metrics, hyperparameters and files) from your Git repository. If
+   your repository has more than 200 columns, you should
+   [select the columns that are mandatory to import](/doc/studio/user-guide/views/view-settings#mandatory-columns).
+   DVC Studio will also import the unselected columns, but only up to a maximum
+   of 200 columns.
+
+   **What if there are more than 200 mandatory columns?**<br>Currently DVC
+   Studio cannot import over 200 columns. If you have a large repository (with
+   more than 200 mandatory columns), one solution is to create a mono repo - if
+   your ML project can be divided into sub-projects, create a
+   [monorepo](https://en.wikipedia.org/wiki/Monorepo) where each sub-project
+   resides in a separate directory in the monorepo. Then, for each sub-project,
+   create a view.<br><br>To create views in the case of a monorepo,
+   [specify the project directory in view settings](/doc/studio/user-guide/views/view-settings#project-directory).<br><br>If
+   this solution does not work for your use case, please create a support ticket
```

Comment:
Why do we want custom new lines? Anyway, in Markdown you leave 2 trailing spaces in a line to indicate a forced new line. For 2 new lines, well that's just a new paragraph 🙂 :
```suggestion
   **What if there are more than 200 mandatory columns?**  
   Currently DVC Studio cannot import over 200 columns. If you have a large repository (with
   more than 200 mandatory columns), one solution is to create a mono repo - if
   your ML project can be divided into sub-projects, create a
   [monorepo](https://en.wikipedia.org/wiki/Monorepo) where each sub-project
   resides in a separate directory in the monorepo. Then, for each sub-project,
   create a view.

   To create views in the case of a monorepo,
   [specify the project directory in view settings](/doc/studio/user-guide/views/view-settings#project-directory).
   
   If this solution does not work for your use case, please create a support ticket
```

Reply:
Fixed. Thanks.

Reply:
> Do you need someone else from the Studio team to review this for accuracy

Yes, just requested @duijf